### PR TITLE
autopopulate values in data constraints

### PR DESCRIPTION
**Pull request by khushbu-maurya**

### CODE REVIEW

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

File: src/main/webapp/app/entities/flair-bi/modal/modal-tabs/condition/condition.component.js [hunk 1]
```diff
@@ -51,7 +51,9 @@
         }
 
         function getMetadataTooltip(metadata) {
-            return 'from ' + metadata.startDateFormatted + ' to ' + metadata.endDateFormatted;
+            if(metadata){
+                return 'from ' + metadata.startDateFormatted + ' to ' + metadata.endDateFormatted;
+            }
```
Comment (ir73) [inline src/main/webapp/app/entities/flair-bi/modal/modal-tabs/condition/condition.component.js]: a function should always return a value, so perhaps you can add `return ''` in the bottom

Comment (khushbu-maurya) [reply src/main/webapp/app/entities/flair-bi/modal/modal-tabs/condition/condition.component.js]: okay

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

**Pull request merged**